### PR TITLE
Changed the section name for the timer class to be in the start method

### PR DESCRIPTION
**Pull request by L7ColWinters**
This follows the more standard android.os.Trace class which allows to make traces or also use Firebase performance monitoring with an overrode DebugTimer class in a local project.

### CODE REVIEW

File: epoxy-adapter/src/main/java/com/airbnb/epoxy/DebugTimer.java [hunk 1]
```diff
@@ -14,10 +15,11 @@
 
   private void reset() {
     startTime = -1;
+    sectionName = null;
   }
 
   @Override
-  public void start() {
+  public void start(String sectionName) {
```
Comment (elihart) [inline epoxy-adapter/src/main/java/com/airbnb/epoxy/DebugTimer.java]: this isn't used, did you mean to assign the field?

Comment (L7ColWinters) [reply epoxy-adapter/src/main/java/com/airbnb/epoxy/DebugTimer.java]: whoops, was copying over from other repo

**Comment by L7ColWinters** [comment]
It would be nice to have an API but then on every EpoxyController it would need to pass in an optional Timer instance. What i'm doing is overriding the DebugTimer class under the airbnb package to inject my own implementation. One could also make a static method to set the instance but I'm not a fan of static methods for DI. Do you think it would make sense to add a static method on the EpoxyController class?

**Comment by elihart** [comment]
I would be fine with a static method to set a global change to the Timer, we already have some other static methods for global defaults.

You're right that it's not ideal, but imo it's better than overriding the class.

**Pull request merged**